### PR TITLE
added events SDL_DisplayEvent and SDL_SensorEvent from last SDL2 2.0.9

### PR DESCRIPTION
**Pull request by fccm**

### CODE REVIEW

**Comment by dbuenzli** [comment]
Thanks @fccm for your PR. I don't have the time to get into this right now. I will tentatively ask past contributors @Octachron or @tokenrove (feel free to ignore :-) to have a quick look at this so that it can be merged.

File: src/tsdl.mli [hunk 1]
```diff
@@ -3033,6 +3033,30 @@ module Event : sig
   val user_window_id : int field
   val user_code : int field
 
+  (** {2:display Display events} *)
+
+  val display_event : event_type
+
+  (** {3 {{:http://wiki.libsdl.org/SDL_DisplayEvent}SDL_DisplayEvent} fields} *)
```
Comment (tokenrove) [inline src/tsdl.mli]: This page doesn't exist yet, but presumably will by the time 2.0.9 is released?

**Comment by fccm** [comment]
The FIXME comment is copy/pasted from an existing one at line 3266 where the structure "SDL_JoystickGUID" has a similar array.

**Comment by dbuenzli** [comment]
Thanks Florent your patches are in. And thanks @tokenrove for the review !

**Pull request closed without merging**